examples/autoutils/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../common.cmake")
+
+project(download-autoutils)
+
+# download autoutils
+hunter_add_package(autoutils)
+
+if(NOT EXISTS "${autoutils_LICENSES}")
+  message(FATAL_ERROR "File not found: '${autoutils_LICENSES}'")
+endif()
+
+message("License: '${autoutils_LICENSES}'")
+
+# now autoutils can be used
+# DOCUMENTATION_START {
+list(APPEND CMAKE_MODULE_PATH "${AUTOUTILS_ROOT}/cmake")
```

Comment:
You can generate `*Config.cmake` with this line so it will use common `hunter_add_package`/`find_package(... CONFIG)` pattern. See example:
* https://github.com/hunter-packages/gauze/blob/b4bc883937e9cba8b8b9bbb32532fd07c3446eda/cmake/Config.cmake.in